web/client/translations/data.en-US

```diff
@@ -94,7 +94,7 @@
             "warningSaveUpdatedMap": "Some layers haven't been updated correctly",
             "saveUpdatedMap": "All the layers have been successfully updated",
             "incompatibleBackgroundAndProjection":"The Projection you selected is not compatible with background, switch to a compatible -or an empty- background, then select this projection!",
-            "incompatibleDataAndProjection":"the current layer and map projection are not completely compatible. Parts or all of the data might not appear in the map"
+            "incompatibleDataAndProjection":"the current layer/s and map projection are not completely compatible. Parts or all of the data might not appear in the map"
```

Comment:
```suggestion
            "incompatibleDataAndProjection":"the current layer and map projection are not completely compatible. Parts or all of the data might not appear in the map"
```

Reply:
I think it was a typo, please always take a look to the pull request's changes on  your own to find out if you committed something wrong

Reply:
Sorry, @tdipisa explained you meant to say something like `layer(s)`.